manifests/03_credentials_request_azure_file.yaml

```diff
@@ -31,7 +31,9 @@ spec:
       - 'Microsoft.Network/virtualNetworks/subnets/*/read'
```

Comment:
@RomanBednar Better to also remove `'Microsoft.Network/virtualNetworks/subnets/*/read'`, it is duplicated with `'Microsoft.Network/virtualNetworks/subnets/read'`.
I filed upstream PR correct it ->
https://github.com/kubernetes-sigs/azurefile-csi-driver/pull/2557

Reply:
@Phaow Good catch, I reflected the upstream change here.